frontend/src/metabase/query_builder/components/filters/modals/InlineCategoryPicker/InlineCategoryPicker.styled.tsx

```diff
@@ -0,0 +1,27 @@
+import styled from "@emotion/styled";
+import { color } from "metabase/lib/colors";
+import { space } from "metabase/styled-components/theme";
+
+import LoadingSpinner from "metabase/components/LoadingSpinner";
+
+export const Loading = styled(LoadingSpinner)`
+  margin: ${space(1)} 0;
+  color: ${color("brand")};
+`;
+
+export const PickerContainer = styled.div`
+  grid-column: span 2 / span 2;
```

Comment:
Cool. I haven't used grid much, but how is this different from just `grid: span 2`? I just noticed tailwind also use this style of `span x / span x`.

Reply:
that's a good question... I copied it from the tailwind docs 😂